### PR TITLE
committing to add reviewer

### PR DESCRIPTION
**Pull request by michael-greene-ey**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "./index.css";
 import App from "./App";
 import "bootstrap/dist/css/bootstrap.min.css";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+const root = ReactDOM.createRoot(document.getElementById("root")!);
```
Comment (ninohero) [inline src/index.tsx:7]: do you really want that negation '!' ?

**Review by ninohero — COMMENTED** [review]
just one comment.  that '!' seems weird.  does it not break anything?

File: src/index.tsx (hunk 1 above)
Comment (michael-greene-ey) [reply src/index.tsx:7]: it tells ts that line 7 will always return an element and never return null. Therefore it doesn't break anything.

**Comment by ninohero** [comment]
looks good to me!

**Pull request merged**